### PR TITLE
Cleanup Utility to remove stale DBB objects and build libraries

### PR DESCRIPTION
**Pull request by dennis-behm**
This common backend script allows to perform automated cleanup of the DBB Collections and Build Groups and the Build datasets.

This implements #254

### CODE REVIEW

File: Templates/Common-Backend-Scripts/README.md [hunk 1]
```diff
@@ -887,7 +887,112 @@ It _greps_ the information and invokes a download action.
...
+---------- | ----------------------------------------------------------------------------------------
+-w `<workspace>` | **Workspace directory** - an absolute or relative path that represents unique directory for this pipeline definition, that needs to be consistent through multiple steps. 
+-a `<application>` | **Application name** to be analyed for stale DBB Metadatastore objects and Build Datasets.
+-p | Flag to control if the generated delete cmd files should be executed by the pipeline. If the steps are not executed through the script, it is recommended to publish the generated files to the pipeline run, that an administrator can take care of them.
+
+#### Additional Notes
+
+This script can be embedded into a pipeline execution, but can also be used in a stand-alone setup. For a pipeline implementation, this task can be included in the release process to facilate the cleanup of stale DBB Collections and DBB Build Groups, as well to make delete the build datasets.
+
+For the standalone implementation follow the below process:
```
Comment (M-DLB) [inline Templates/Common-Backend-Scripts/README.md:914]:
```suggestion
For the standalone implementation, use the following process:
```

File: Templates/Common-Backend-Scripts/README.md [hunk 2]
```diff
@@ -887,7 +887,112 @@ It _greps_ the information and invokes a download action.
...
+1. Have the Common Backend Scripts installed to z/OS Unix System Services and have them configured. 
+2. Clone the application repository including all remote references.
+3. Assemble the generateCleanupCommands command like in the above sample. The user executing the script needs proper permissions on the DBB Metadatastore.
```
Comment (M-DLB) [inline Templates/Common-Backend-Scripts/README.md:917]:
```suggestion
3. Execute the `generateCleanupCommands.sh` script like in the above sample. The user executing the script needs proper permissions on the DBB Metadatastore.
```

**Review by M-DLB — COMMENTED** [review]
Some suggestions

**Comment by dennis-behm** [comment]
@M-DLB Thanks for the review and suggestions. Please see the update in https://github.com/IBM/dbb/pull/255/commits/6a023ca7c1c78aa9acd62d864c0f80bb5dcd14f6

**Review by M-DLB — APPROVED** [review]
Thank you @dennis-behm

**Pull request merged**